Content.Server/_CD/Loadouts/RenameIdComponent.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Content.Server._CD.Loadouts;
+
+/// <summary>
+/// Marker that should be attached to the PDA to rename the contained ID to the user's requested job. Used to implement custom tile loadouts.
+/// </summary>
+[RegisterComponent]
+public sealed partial class RenameIdComponent : Component
+{
+    [DataField]
+    public string Value;
```

Comment:
```suggestion
    public LocId Value;
```
validates loc IDs in yaml.